dm/syncer/syncer.go

```diff
@@ -1618,6 +1618,9 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 	if err != nil {
 		return err
 	}
+	if !fresh && s.cfg.SafeModeDuration == "0s" {
```

Comment:
Why not fresh and no safemode means error? If we have safemode exit point, we don't need safemode